### ISSUE TITLE
Break world into 'screen chunks' 

### ISSUE THREAD

**Issue by MondayHopscotch** [bitDecayGames/Jump#62]
This might solve both the localized updates issue mentioned in #54 and the problem of things getting out of sync. 

If we break the level unto 'chunks' that are of some arbitrary size (will probably need to be based on the desired camera viewport so we are always updating what is on screen) and always update an entire chunk - we now have eliminated any issues with synchronization. 

Though if we have sections of levels that are larger than 1 chunk, we sill have this issue. 

Food for thought. We don't need to address it quite yet.

**Comment by Kenoshen** [comment]
Moving platform that spans the entire level, such as with many Mario games.  Seems like this might not work?  I don't know though.

**Comment by MondayHopscotch** [comment]
Yep. Those most certainly would have issues even with this approach... damn it. I guess the options from there would be to either design around that limitation, or spend time coming up with a way of addressing it.
